bin/runFile.js

```diff
@@ -1,23 +1,27 @@
 const path = require('path');
 const util = require('util');
 const taiko = require('../lib/taiko');
+
 const { removeQuotes, symbols } = require('../lib/util');
-module.exports = (file, observe, observeTime) => {
+module.exports = async (file, observe, observeTime, continueRepl) => {
     const realFuncs = {};
     for (let func in taiko) {
         realFuncs[func] = taiko[func];
         if (realFuncs[func].constructor.name === 'AsyncFunction')
             global[func] = async function () {
                 let res, args = arguments;
-                if (func === 'openBrowser' && observe) {
+                if (func === 'openBrowser' && (observe || continueRepl)) {
                     if (args['0']) {
```

Comment:
do we wanna open browser in headful mode for load always ? if so when user has `args['0']` set and observe is not used it will be `headless: true` (!undefined)

either we can set observe true for load by default or let user decide if it has to be in observe mode by explicitly saying it

Reply:
> do we wanna open browser in headful mode for load always ?

I would prefer it to be, coz we always launch REPL in headful mode. If run script it headless there won't be a way to bring the browser in foreground.
> either we can set observe true for load by default

This can be done.